actions/emoji_actions.test.js

```diff
@@ -4,7 +4,7 @@
 import configureStore from 'store';
 import {getRecentEmojis, getEmojiMap} from 'selectors/emojis';
 import * as EmojiActions from 'actions/emoji_actions';
-import LocalStorageStore from 'stores/local_storage_store';
+import preferences from 'mattermost-redux/action_types/preferences';
```

Comment:
Shouldn't this import look more like `import * as PreferenceActions from 'mattermost-redux/actions/preferences';`? You're spying on a set of constants currently